Dockerfile

```
@@ -8,14 +8,14 @@ WORKDIR DbTools/
 RUN dotnet build ./DbTools.csproj -c Release -o /app_tools
 
 # Comment in the following line for local development
-# RUN mkdir -p /DbTools/bin/Debug/net9.0 && cp /app_tools/DbTools /DbTools/bin/Debug/net9.0/DbTools
+# RUN mkdir -p /DbTools/bin/Debug/net10.0 && cp /app_tools/DbTools /DbTools/bin/Debug/net10.0/DbTools
 
 WORKDIR ../Storage/
```

Hadolint (2.14.0)

[error] Use absolute WORKDIR

(DL3000)